.ado/pipelines/templates/stages-loadtest-locust.yaml

```diff
@@ -9,20 +9,11 @@ parameters:
    default: ''
```

Comment:
as discussed: how about putting this directly into the pipeline file? that is the only place it is now referenced, right?

Reply:
will be part of the next PR probably next week